Doc/library/stdtypes.rst

```diff
@@ -1794,8 +1796,8 @@ expression support in the :mod:`re` module).
    Return ``True`` if the string is a valid identifier according to the language
    definition, section :ref:`identifiers`.
 
-   Call :func:`keyword.iskeyword` to test whether string ``s`` is a reserved
-   identifier, such as :keyword:`def` and :keyword:`class`.
+   :func:`keyword.iskeyword` can be used to test whether string ``s`` is a
+   reserved identifier, such as :keyword:`def` and :keyword:`class`.
```

Comment:
```suggestion
   :func:`keyword.iskeyword` can be used to test whether string ``s`` is a reserved
   identifier, such as :keyword:`def` and :keyword:`class`.
```

Reply:
Huh? I don't see how this minimizes the diff, since `reserved` wasn't there before, and rather just needlessly splices "reserved identifier" and leads to less semantically meaningful breaks, which _increases_ future diff noise (by increasing the change that changes to a single atomic semantic unit touch multiple lines)

Reply:
You're incorrect, `reserved` was there before. Take a look at the [final commit diff](https://github.com/python/cpython/commit/a0d940d6acbb5c6614cf892192d8cb0d7002e5a6).

Reply:
I think going for semantically meaningful line breaks is fine as a general goal, but I also value git history. In cases like these, which fall under the category of "minor copyedits" rather than "substantial rewrite", I prefer to follow the longstanding policy of keeping the diff as minimal as possible in order to preserve git history.